vendor/src/github.com/docker/libnetwork/ipams/remote/remote_regular.go

```diff
@@ -0,0 +1,28 @@
+// +build !experimental
```

Comment:
I thought so too. But plugins will exit experimental shortly. Hence having this code duplication isnt a big problem (given the fact that it will be converged before 1.13 release).